serverless/functions/Security.yml

```diff
@@ -0,0 +1,59 @@
+# ================== Security =====================
+
+securityInitialization:

```

Comment:
I suggest to use `api` prefix for name. In logs you will see `apiSecurityInititalization` and known this an API method.